src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
```

Comment:
Add parenthesis and line break:

When using complex comparisons, with &&, ||, add parenthesis between equation parts so it's easier to read.
It's also good to add line breaks if the line is took long. Here is an example:
![image](https://github.com/pedro-ruas/mate-v2-js_average-ages/assets/54222238/eb906b28-bd89-40a2-bcb1-9f0fd53e50fa)